src/app/components/pokemons/pokemons.component.html

```diff
@@ -0,0 +1,27 @@
+<div class="general-content" infiniteScroll [infiniteScrollDistance]="0.2" (scrolled)="onScroll()">
+  <div class="pokemons-container">
+    <div *ngFor="let pokemon of pokemons; index as i" class="pokemon-container">
+      <div class="image-container">
+        <img class="pokemon-image" src="{{imgUrl + pokemon.index}}.png" (click)="open(pokemon.name)"
```

Comment:
use input instead of interpolation, if there is no input for the attribute use the generic input [attr.attributename]